server/common/oursrc/scripts-proxy/main.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"net"
+	"strings"
+
+	"inet.af/tcpproxy"
+)
+
+var (
+	httpAddrs   = flag.String("http_addrs", ":80", "comma-separated addresses to listen for HTTP traffic on")
```

Comment:
Does [::]:443 syntax work here? Seeing no address at all confuses me.

Reply:
Empty address is the same as 0.0.0.0. This is standard for Go (see e.g. the default of ":http" for the Addr field in https://pkg.go.dev/net/http#Server)

Reply:
-1 do not like this convention.

Reply:
Ookay. Well since we're unlikely to have IPv6 anytime soon, I've made the default have 0.0.0.0 instead. The standard library treats "" as the address-family-specific wildcard address, so that really is well-defined even if you don't like it.